test/ServerIntegrationTest.php

```diff
@@ -1,49 +0,0 @@
-<?php
-/**
- * @see       https://github.com/zendframework/zend-diactoros for the canonical source repository
- * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
- * @license   https://github.com/zendframework/zend-diactoros/blob/master/LICENSE.md New BSD License
- */
-
-namespace ZendTest\Diactoros;
-
-use PHPUnit\Framework\TestCase;
-use Prophecy\Argument;
-use Psr\Http\Message\ResponseInterface;
-use Psr\Http\Message\ServerRequestInterface;
-use Zend\Diactoros\Response\SapiStreamEmitter;
-use Zend\Diactoros\Server;
-
-class ServerIntegrationTest extends TestCase
-{
-    public function testPassesBufferLevelToSapiStreamEmitter()
```

Comment:
This test was added as part of the patch around the argument swap in `SapiStreamEmitter`, and thus became unnecessary once we stopped working with the output buffer.